server/src/server.ts

```diff
@@ -1,21 +1,170 @@
 import express, { Application } from 'express';
 import cors from 'cors';
+import cookieSession from 'cookie-session';
 import dotenv from 'dotenv';
-import recordRouter from './routes/record';
-import { connectToServer } from './db/connection';
-
+import route from './routes/registerRoute'
+import routes from './routes/loginRoute';
+import routee from './routes/currentUser'
+import { connect } from './db/connection';
+import { createUser, getEmail, getUsers, User, workoutModel } from './model';
+import bcrypt from 'bcrypt';
 dotenv.config({ path: './config.env' });
+import jwt from 'jsonwebtoken'
+const secretKey: string = process.env.SECRET_KEY || '';
+
+
 
 const app: Application = express();
 const port: string | number = process.env.PORT || 7036;
 
 app.use(cors());
+app.use(routes);
```

Comment:
Consider giving more meaningful names to these routes after making the changes I requested previously